### PR TITLE
Change "claim" to "register" and fix updateAvailable grammar 

### PR DESCRIPTION
**Pull request by shapkarin**

### CODE REVIEW

**Comment by shapkarin** [comment]
Please, help to change the other translates.

**Review by mradkov — CHANGES_REQUESTED** [review]
Please remove the updated `package-lock.json` from this PR.

**Review by mradkov — DISMISSED** [review]
It looks good now. Thanks.

**Review by mradkov — CHANGES_REQUESTED** [review]
Can you rebase to develop and resolve conflicts?

**Comment by shapkarin** [comment]
@mradkov it's already rebased and conflict was fixed

**Comment by mradkov** [comment]
> @mradkov it's already rebased and conflict was fixed

Screenshot 2020-09-15 at 18 00 00

**Comment by mradkov** [comment]
Can you rebase on develop instead of merge into your branch, please?

File: src/popup/locales/en.json [hunk 1]
```diff
@@ -299,8 +299,8 @@
       "backup": "backup",
       "yourSeedPhrase": "your seed phrase",
       "updateExtension": "New version available! Please update your extension.",
-      "updateAvailable": "New version available in the store!",
-      "claim-name": "Claim your .chain name"
+      "updateAvailable": "New version in store available!",
```
Comment (shapkarin) [inline src/popup/locales/en.json:302]:
```suggestion
      "updateAvailable": "New version in the store is available!",
```

**Comment by mradkov** [comment]
> Can you rebase on `develop` instead of merge into your branch, please?
> 
> Be sure you've read and are sticking to the [contribution guidelines](https://github.com/aeternity/superhero-wallet/blob/develop/CONTRIBUTING.md) cc @shapkarin

@shapkarin ^

**Comment by shapkarin** [comment]
@mradkov OK, I will check that closely

**Comment by shapkarin** [comment]
@mradkov Please, check. I hope that now it's fine.

**Comment by shapkarin** [comment]
@mradkov I removed test case `Account info contain claim name message`. Not sure that it needs to be removed or changed.

File: src/popup/locales/ru.json [hunk 2]
```diff
@@ -231,8 +231,8 @@
       "backup": "backup",
       "yourSeedPhrase": "your seed phrase",
       "updateExtension": "New version available! Please update your extension.",
-      "updateAvailable": "New version in store available!",
-      "claim-name": "Claim your .chain name"
+      "updateAvailable": "New version in the store is available!",
+      "claim-name": "Register the name"
```
Comment (mradkov) [inline src/popup/locales/ru.json:235]: Shouldn't it be `Register a name` instead?

Comment (shapkarin) [reply src/popup/locales/ru.json:235]: Yes, it should. Because no there no any current name before it's defined.

**Comment by mradkov** [comment]
> @mradkov I removed test case `Account info contain claim name message`. Not sure that it needs to be removed or changed.

Can you bring it back and adapt it to the current changes in this PR?

**Review by mradkov — APPROVED** [review]
Looks good now. Thanks!

**Pull request merged**